coral/src/app/features/browse-topics/hooks/topic-list/useGetTopics.test.tsx

```diff
@@ -159,11 +176,25 @@ describe("useGetTopics", () => {
       expect(result.current.data).toMatchObject(
         mockedResponseMultiplePageTransformed
       );
+
+      const apiCallUrls = getSpyCallUrls(spyGet);
+      expect(apiCallUrls.length).toBe(1);
+      expect(apiCallUrls[0].pathname).toEqual("/getTopics");
+      expect(apiCallUrls[0].searchParams.get("pageNo")).toEqual("2");
+      expect(apiCallUrls[0].searchParams.get("env")).toEqual("ALL");
+      expect(apiCallUrls[0].searchParams.get("teamName")).toEqual(null);
     });
 
-    it("returns a list of topics with current page set to 3", async () => {
+    it("when called with a page number", async () => {
```

Comment:
Small nitpick: I would write that `it` block in the same style as the others, something like "includes 'pageNo' as a query parameter in the API request"